<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="fuelGauge">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/fuelguage.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="tachometer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">r gb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/tachometer1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="rightInd">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>350</y>
      <width>35</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/turn_indicator.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="leftInd">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>35</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/turn_indicator_left.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="speedometer">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>40</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/odometer.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
